python/lsst/sims/maf/driver/runBundle.py

```diff
@@ -0,0 +1,96 @@
+import os, re, copy, warnings
+import lsst.sims.maf.sliceMetrics as sliceMetrics
+import lsst.sims.maf.db as db
+import lsst.sims.maf.utils as utils
+
+def runBundle(mafBundle, verbose=True, makePlots=True):
+    """
+    mafBundle should be a dict with the following key:value pairs:
+    'metricList':List of MAF metric objects
+    'slicer': instatiated MAF slicer
+    'dbAddress': string that gives the address to the desired DB
```

Comment:
We're making them pass other instantiated objects, so it would be better to make them instantiate the connection to the DB. That way we don't have to worry about non-opsim databases, and it's a simple enough thing to instantiate. (plus we probalby don't want to re-instantiate this for each 'bundle'.)